user_interaction/static/themes/dark-theme.css

```diff
@@ -40,13 +40,34 @@
     --greenify-dropdown-hover-text-color: inherit;
     --greenify-dropdown-active-bg-color: var(--greenify-bg-color);
     --greenify-dropdown-active-text-color: var(--greenify-text-color);
+
+    /*Calendar day color*/
+    --fc-neutral-bg-color: green;
 }
 
 /* Lighten link colors */
 a {
     color: #55a7ff;
 }
+a:hover {
```

Comment:
The same fix would be nice in the Dining theme

Reply:
done